bin/alluxio-monitor.sh

```diff
@@ -56,10 +56,10 @@ get_env() {
 
   # Remove the remote debug configuration to avoid the error: "transport error 20: bind failed: Address already in use."
   # See https://github.com/Alluxio/alluxio/issues/10958
```

Comment:
add the issue url https://github.com/Alluxio/alluxio/issues/15168 to this comment?

Reply:
Done